.ci/pytorch/build.sh

```diff
@@ -241,13 +246,18 @@ if [[ "$BUILD_ENVIRONMENT" == *-bazel-* ]]; then
     tools/bazel build --config=no-tty "${BAZEL_MEM_LIMIT}" "${BAZEL_CPU_LIMIT}" //...
   fi
 else
+  git config --global --add safe.directory /var/lib/jenkins/workspace
+
   # check that setup.py would fail with bad arguments
   echo "The next three invocations are expected to fail with invalid command error messages."
   ( ! get_exit_code python setup.py bad_argument )
   ( ! get_exit_code python setup.py clean] )
   ( ! get_exit_code python setup.py clean bad_argument )
 
   if [[ "$BUILD_ENVIRONMENT" != *libtorch* ]]; then
+    pushd ..
+    sudo chown -R jenkins workspace
+    popd
```

Comment:
We could save 2 lines of code by making this do `sudo chown -R jenkins ../workspace`